ci-operator/config/openshift/driver-toolkit/openshift-driver-toolkit-master.yaml

```diff
@@ -12,6 +16,13 @@ images:
   dockerfile_path: Dockerfile
   from: base
   to: driver-toolkit
+- dockerfile_literal: |
```

Comment:
You could just add this in your Dockerfile for the `root` image.

Reply:
Ah ignore me. I just saw that you are not building your root. https://github.com/openshift/driver-toolkit/blob/master/.ci-operator.yaml

Reply:
Done. Thank you.